### PR TITLE
feat(module:color-picker): make color picker standalone

### PR DESCRIPTION
**Pull request by Nicoss54**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Color Picker component is not standalone

Issue Number: N/A

## What is the new behavior?

Color picker component is standalone

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by HyperLife1119 — APPROVED** [review]
LGTM

File: components/color-picker/color-block.component.ts [hunk 1]
```diff
@@ -5,14 +5,16 @@
 
 import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
 
-import { defaultColor } from 'ng-antd-color-picker';
+import { defaultColor, NgAntdColorPickerModule } from 'ng-antd-color-picker';
```
Comment (HyperLife1119) [inline components/color-picker/color-block.component.ts:8]: NIT: @OriginRing 我们可以完全集成 ng-antd-color-picker 到 zorro 吗？而不是将它作为外部依赖。

Comment (OriginRing) [reply components/color-picker/color-block.component.ts:8]: Absolutely, I will deal with this issue in January @HyperLife1119

**Review by OriginRing — APPROVED** [review]

**Comment by OriginRing** [comment]
LGTM

**Pull request merged**